apps/davi/src/components/ProposalVoteCard/ProposalVoteCard.tsx

```diff
@@ -60,7 +62,23 @@ const ProposalVoteCard = ({
     [proposal, timestamp]
   );
 
-  const voteOnProposal = useVoteOnProposal(votingMachineAddress);
+  // TODO: implement scheme selector/fetcher
+  // ! For now it is hardcoded untill we implement a scheme selector / fetcher
+  const schemeAddress = useMemo(() => {
+    if (governanceImplementationName === 'Governance1_5') {
+      return '0x2b3edbada0f3f42ac547666b1c6e0b26b2a6ac51';
+    } else {
+      return null;
+    }
+  }, [governanceImplementationName]);
+
+  const votingMachine =
+    useGetVotingMachineAddressBySchemeAddress(schemeAddress);
+
+  const voteOnProposal = useVoteOnProposal(
+    guildId,
+    votingMachine?.votingMachineAddress
+  );

```

Comment:
Can we move the logic of `useGetVotingMachineAddressBySchemeAddress` to the `useVoteOnProposal` hook?

That would avoid implementing another hook for just one of the implementations.